pyfr/plugins/fluidforce.py

```diff
@@ -109,7 +151,7 @@ def __call__(self, intg):
 
             # Interpolate to the face
             ufpts = np.dot(m0, uupts.reshape(nupts, -1))
-            ufpts = ufpts.reshape(nfpts, self.nvars, -1)
+            ufpts = ufpts.reshape(nfpts, nvars, -1)
             ufpts = ufpts.swapaxes(0, 1)
```

Comment:
I tried to confirm the previous implementation (one line of np.dot()) uses BLAS, but it was not easy for me. As an alternative, I checked timeit of this and the previous implementation and found that the current one is much faster. Thus, I rolled back these lines and fixed line 174~175 to multiply m4 matrix to uupts.